gf-button/README.md

```diff
@@ -34,81 +33,78 @@ GFButton(
 ),
 ```
 
-GF Buttons can be styled with several attributes to look a specific way. Each and every attribute is described below.
+## Propiedades
 
-## Properties 
+Se pueden utilizar muchos atributos para darle estilo a los botones GFButton y que luzcan de una manera espec&iacute;fica. Cada uno de los atributos se describe a continuaci&oacute;n.
 
-GF Buttons can be styled with several attributes to look a specific way. Each and every attribute is described below.
+### Color
 
-### Color 
-
-GFColor is used to change the background of the button.
+GFColor es utilizado para cambiar el color de fondo del bot&oacute;n.
 
 |  |  |
 | :--- | :--- |
-| **Description** | The color to use from application's color palette                                |
-| **Attribute** |  color |
-| **Type** | `GFColor` |
```

Comment:
I added an "s", it should be GFColors.

Reply:
Can you explain Where exactly you added "S"

Reply:
OK understand, update it Thanks.